### PR TITLE
Add libraries page for java

### PR DESCRIPTION
**Pull request by svrnm**
I just saw that https://opentelemetry.io/docs/instrumentation/java/libraries/ just says "to be done". I don't want to remove that page, so I tried to come up with a quick fillder, unfortunately I am not 100% sure if & how instrumentation libraries can be used outside of the java agent, maybe some one can help me with that, cc @open-telemetry/java-approvers , @open-telemetry/java-instrumentation-approvers 

Preview: https://deploy-preview-3289--opentelemetry.netlify.app/docs/instrumentation/java/libraries/

### CODE REVIEW

File: content/en/docs/instrumentation/java/libraries.md [hunk 1]
```diff
@@ -4,4 +4,46 @@ linkTitle: Libraries
...
+    <dependency>
+        <groupId>io.opentelemetry.instrumentation</groupId>
+        <artifactId>opentelemetry-grpc-1.6</artifactId>
+</dependency>
+</project>
+```
+
+{{< /tab >}} {{< /tabpane>}}
+
+_tbd_
```
Comment (svrnm) [inline content/en/docs/instrumentation/java/libraries.md:49]: here I need some help

**Review by theletterf — COMMENTED** [review]
Some rewording suggestions.

**Comment by svrnm** [comment]
@open-telemetry/java-approvers @open-telemetry/java-instrumentation-approvers please take a look

File: content/en/docs/instrumentation/java/libraries.md [hunk 2]
```diff
@@ -4,4 +4,22 @@ linkTitle: Libraries
...
+accelerate your work and avoid duplicated efforts. If you instrument your app
+with OpenTelemetry, you don't want to spend additional time on manually adding
+traces, logs, and metrics to those libraries and frameworks.
+
+Use libraries that come with OpenTelemetry support natively or an
+[Instrumentation Library](/docs/concepts/instrumentation/libraries/) to generate
+telemetry data for a library or framework.
+
+The Java agent for automatic instrumentation includes instrumentation libraries
+for many common Java frameworks. All instrumentations are turned on by default.
```
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/instrumentation/java/libraries.md:17]: > > All instrumentations are turned on by default
> 
> this statement is not true

can you provide me with a correct statement to update this accordingly?

[the remark this answers is not in the document]
Comment (mateuszrzeszutek) [reply content/en/docs/instrumentation/java/libraries.md:17]: There's some info here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#disabled-instrumentations

File: content/en/docs/instrumentation/java/libraries.md [hunk 3]
```diff
@@ -4,4 +4,22 @@ linkTitle: Libraries
...
+If you need to turn off certain instrumentation libraries, you can
+[suppress them](../automatic/agent-config/#suppressing-specific-auto-instrumentation).
+
+If you use [manual instrumentation](../manual) for your code, you can leverage
+some instrumentation libraries for your dependencies standalone. To find out,
+which standalone instrumentation libraries are available, take a look at
+[this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
```
Comment (mateuszrzeszutek) [inline content/en/docs/instrumentation/java/libraries.md:24]: I wish we migrated that list over to this repo at some point...

**Review by mateuszrzeszutek — APPROVED** [review]
LGTM

File: content/en/docs/instrumentation/java/libraries.md (hunk 3 above)
Comment (svrnm) [reply content/en/docs/instrumentation/java/libraries.md:24]: +1, yes this page contains a lot of value that could/should live on this particular page. But we can migrate gradually, let's have a basic version of this page first, such that people find the right info, next we can consider moving material.

File: content/en/docs/instrumentation/java/libraries.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (svrnm) [reply content/en/docs/instrumentation/java/libraries.md]: I replaced "all" with most, for now I don't want to make things much more complicated with linking out (see my other comment @ https://github.com/open-telemetry/opentelemetry.io/pull/3289#discussion_r1331724530)

File: content/en/docs/instrumentation/java/libraries.md (hunk 3 above)
Comment (mateuszrzeszutek) [reply content/en/docs/instrumentation/java/libraries.md:24]: Yeah, definitely. I think this PR is a good first step towards that goal.

File: content/en/docs/instrumentation/java/libraries.md [hunk 4]
```diff
@@ -4,4 +4,22 @@ linkTitle: Libraries
...
+[Instrumentation Library](/docs/concepts/instrumentation/libraries/) to generate
+telemetry data for a library or framework.
+
+The Java agent for automatic instrumentation includes instrumentation libraries
+for many common Java frameworks. Most are turned on by default. If you need to
+turn off certain instrumentation libraries, you can
+[suppress them](../automatic/agent-config/#suppressing-specific-auto-instrumentation).
+
+If you use [manual instrumentation](../manual) for your code, you can leverage
+some instrumentation libraries for your dependencies standalone. To find out,
```
Comment (trask) [inline content/en/docs/instrumentation/java/libraries.md:22]:
```suggestion
some instrumentation libraries for your dependencies standalone. To find out
```

**Review by trask — APPROVED** [review]

**Comment by svrnm** [comment]
Outsch .. now I screwed up the submodules myself :-/

**Pull request merged**